.gitignore

```diff
@@ -1,2 +1 @@
-node_modules/
-package-lock.json
```

Comment:
it's needed to see which exact packages was used at the time of bundling, as well as the tools that bundled it